client/state/sites/selectors/has-opt-out-new-stats-notice.js

```diff
@@ -0,0 +1,18 @@
+import 'calypso/state/ui/init';
+import getRawSite from 'calypso/state/selectors/get-raw-site';
+
+/**
+ * Returns whether the opt-out notice should be shown.
+ *
+ * @param  {Object}  state  Global state tree
+ * @param siteId The site ID.
+ * @returns {?boolean}        hasOptOutNotice
+ */
+export default function hasOptOutNewStatsNotice( state, siteId ) {
+	if ( ! siteId ) {
+		return null;
+	}
+	const site = getRawSite( state, siteId );
+
+	return site?.has_opt_out_new_stats_notice;
```

Comment:
```suggestion
	return site?.stats_notices?.opt_out_new_stats;
```

I am not sure about the correctness, but I have to change the site config properties to make the notice show.

<img width="241" alt="截圖 2023-02-03 下午3 28 55" src="https://user-images.githubusercontent.com/6869813/216540394-a1b11a6e-6336-4d92-acc5-379e64441355.png">

I found the [returned site config](https://github.com/Automattic/jetpack/pull/28733/files#diff-f2c1a159814b020c8ba57e8123158c5d50c2d078c14b96de2da89fde1d89cf74R252) from the dependent Jetpack changes has the object structure corresponding to my suggestion above.

Reply:
Good catch. I didn't push the changes to remote oops 😄 fixed now. Thanks